app/components/work_packages/date_picker/date_form_component.rb

```diff
@@ -81,7 +81,8 @@ def text_field_options(name:, label:)
           classes: "op-datepicker-modal--date-field #{'op-datepicker-modal--date-field_current' if @focused_field == name}",
           validation_message: validation_message(name),
           type: field_type(name),
-          placeholder: placeholder(name)
+          placeholder: placeholder(name),
+          aria: { live: :polite, atomic: true }
```

Comment:
Sorry to be nit-picky but I really think that the `default_field_options` are the better place for this. The `text_field_options` are basically dependent on the actual field while the `default_field_options` are meant to be the same for all three fields.